packages/block/src/header.ts

```diff
@@ -913,7 +913,7 @@ export class BlockHeader {
     try {
       const header = (await blockchain.getBlock(hash)).header
       return header
-    } catch (error) {
+    } catch (error: any) {
```

Comment:
Where is this coming from? Is this a new linter rule? Or is it a TypeScript version related change?

Reply:
yes it was changed in v4, from my understanding it is a more strict enforcement of `noImplicitAny` (https://github.com/microsoft/TypeScript/issues/36775#issuecomment-651890339)

Reply:
Seems to have been promoted by the Ethereum community. Really strange how small the world sometimes is! 😛

Reply:
(but makes sense, I like that)